online-banking-cz/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by atef on 25/8/2022.
+  -->
+
+<resources>
+
+
+    <style name="AdyenCheckout.Issuer" />
+
+    <style name="AdyenCheckout.Issuer.TermsAndConditions">
+        <item name="android:text">@string/checkout_issuers_list_terms_and_conditions_text</item>
+        <item name="android:minHeight">@dimen/termsAndConditions_minHeight</item>
+        <item name="android:paddingStart">@dimen/termsAndConditions_paddingStart</item>
+        <item name="android:paddingEnd">@dimen/termsAndConditions_paddingEnd</item>
+        <item name="android:textSize">@dimen/termsAndConditions_text_size</item>
+        <item name="android:gravity">center_vertical</item>
+        <item name="android:background">?selectableItemBackground</item>
+    </style>
+
+    <style name="AdyenCheckout.DropdownTextInputLayout.Issuer" parent="AdyenCheckout.DropdownTextInputLayout">
```

Comment:
All styles here should be prefixed by `AdyenCheckout.Issuer`, so `AdyenCheckout.Issuer.DropdownTextInputLayout` not `AdyenCheckout.DropdownTextInputLayout.Issuer`

Also maybe it shouldn't be called `AdyenCheckout.Issuer`, it's only for the online banking CZ component, you can just call it `AdyenCheckout.OnlineBankingCZ`

Reply:
I guess I should have put it like `AdyenCheckout.DropdownTextInputLayout.TermsAndCondition` WDYT?

Reply:
I agree with Atef here. Style names in Android go from global to more specific after each dot. Because `AdyenCheckout.DropdownTextInputLayout` is the base style we want to build upon, we should go more specific from there, so you get something like `AdyenCheckout.DropdownTextInputLayout.OnlineBanking.TermsAndConditions` (or you could choose to omit `OnlineBanking`). This is mostly because of how extending styles works, which also means that if you follow this way you could remove the `parent=...`